.circleci/config.yml

```diff
@@ -704,6 +704,50 @@ step-save-src-cache: &step-save-src-cache
     key: v5-src-cache-{{ arch }}-{{ checksum "src/electron/.depshash" }}
     name: Persisting src cache
 
+# Check for doc only change
+step-check-for-doc-only-change: &step-check-for-doc-only-change
+  run:
+    name: Check if commit is doc only change
+    command: |
+      cd src/electron
+      node script/yarn install --frozen-lockfile
+      export DOC_ONLY_CHANGE="`node script/doc-only-change.js --prNumber=$CIRCLE_PR_NUMBER --prURL=$CIRCLE_PULL_REQUEST`"
+      if [ "$DOC_ONLY_CHANGE" = "true" ]; then
```

Comment:
better to use the return code, e.g.

```bash
if node script/doc-only-change.js ...; then
  # ...
```

Reply:
@nornagon a non zero error code will stop CI execution